app/main/views/g_cloud.py

```diff
@@ -209,7 +209,7 @@ def get_service_by_id(service_id):
         status_code = 200
         if service['serviceMadeUnavailableAuditEvent'] is not None:
             service_unavailability_information = {
-                'date': dateformat(service['serviceMadeUnavailableAuditEvent']['createdAt']),
+                'date': dateformat(framework.get('frameworkExpiresAtUTC')),
```

Comment:
I'm thinking we should either make this fail noisily if there's no `frameworkExpiresAtUTC` field on the framework object and use `framework['frameworkExpiresAtUTC']`, or provide an alternative bit of content in the template (perhaps just `'Supplier X stopped offering this service.'` without the date)? While all our current frameworks have that field, it's not required when we create it - this way we absolutely won't forget to add it 😸

Reply:
Hmm, wouldn't we only find this out when a framework is expired so maybe we would forget to add it initially still?

Reply:
🤔  Normally we'd find out on Preview first (we switch the status there first to sanity check that everything's ok - including old service pages). However I don't think we have any automated coverage for it... Perhaps [an alternative date-less message in the template here](https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/master/app/templates/service.html#L39) is the best bet then.

Reply:
@katstevens OK, I changed it now as per your first suggestion in https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/1050/commits/baad6dd06a5ef1ce54c06cd5f03f2d3c8091b73c.